File: fabonacci.py
class fabonacci_dp():
	def of(self,x):
		dp = [0,1]
		for i in range(x):
			dp.append(dp[i] + dp[i+1])
		print('Fabonacci.of(%d)==%d'%(x,dp[x]))

def main():
	fabonacci = fabonacci_dp()
	for i in range(1,201):
		fabonacci.of(i)

if __name__ == '__main__':
	main()

File: fabonacci/fabonacci.js
class fabonacci_dp{
	of(x){
		var dp = [0,1]
		for(var i=0;i<x;i++){
			dp.push(dp[i]+dp[i+1])
		}
		console.log('Fabonacci(%d)==%d',x,dp[x])
	}
}

function main(){
	fabonacci = new fabonacci_dp()
	for(var i=1;i<=200;i++){
		fabonacci.of(i)
	}
}

main()